doc/contributing.rst

```diff
@@ -15,4 +20,116 @@ Contributions to RSMTool are very welcome. You can use the instructions below to
+        | * Same arguments as RSMTool except the ``skll`` keyword argument is not    |
+        |   applicable.                                                              |
+        +----------------------------------------------------------------------------+
+        | Writing test(s) for RSMPredict                                             |
+        |                                                                            |
+        | * Only positional argument is the name of the test directory you created.  |
+        |                                                                            |
+        | * Use ``excluded=True`` if you want to check the excluded responses file   |
+        |   as part of the test.                                                     |
+        |                                                                            |
+        | * Set ``file_format="tsv"`` (or ``"xlsx"``) if you specified the same      |
+        |   field in the configuration file.                                         |
+        +----------------------------------------------------------------------------+
+        | Writing test(s) for RSMCompare                                             |
+        |                                                                            |
+        | * First positional argument is the name of the test directory you created. |
+        |                                                                            |
+        | * Second positional argument is the comparison ID from the JSON            |
+        |   configuration file.                                                      |
+        +----------------------------------------------------------------------------+
+        | Writing test(s) for RSMSummarize                                           |
+        |                                                                            |
+        | * Only positional argument is the name of the test directory you created.  |
+        |                                                                            |
+        | * Set ``file_format="tsv"`` (or ``"xlsx"``) if you specified the same      |
+        |   field in the configuration file.                                         |
+        +----------------------------------------------------------------------------+
+
+Once you have added all new functional tests, you should run ``nosetests --noglogcapture`` to run all the tests. Obviously, the newly added tests will fail since you have not yet generated the expected output for that test. To do this, you should now run the following:
+
+.. _update_files:
+.. code-block:: text
+    
+    python tests/update_files.py --tests tests --outputs test_outputs
+
+This will copy over the generated outputs for the newly added tests and show you a report of the files that it added. If you run ``nosetests`` again, your newly added tests should now pass. 
+
```

Comment:
"If run correctly, you should only see changes to model files and the files affected by the functionality you implemented"